Source/ZoomNet/Models/PaginatedResponseWithTokenAndDateRange.cs

```diff
@@ -0,0 +1,58 @@
+using Newtonsoft.Json;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Pagination Object.
+	/// </summary>
+	/// <typeparam name="T">The type of records.</typeparam>
+	public class PaginatedResponseWithTokenAndDateRange<T>
+	{
+		/// <summary>
+		/// Gets or sets the start date for the report.
+		/// </summary>
+		/// <value> Start date for this report in 'yyyy-mm-dd' format.</value>
+		public string From { get; set; }
+
+		/// <summary>
+		/// Gets or sets the end date for the report.
+		/// </summary>
+		/// <value> End date for this report in 'yyyy-mm-dd' format.</value>
+		public string To { get; set; }
+
+		/// <summary>
+		/// Gets or sets the number of items returned on this page.
+		/// </summary>
+		/// <value>The number of items returned on this page.</value>
+		[JsonProperty(PropertyName = "page_count")]
+		public int PageCount { get; set; }
```

Comment:
According to Zoom's [documentation](https://marketplace.zoom.us/docs/api-reference/pagination) under the `Replacing Page Numbers with Next Page Token` section, they are in the process of deprecating "page number" and page count" (however their documentation is not 100% consistent: , I have seen other documentation pages where they show the page count property). Therefore we should remove this property.